index.html

```diff
@@ -128,7 +128,7 @@ <h3>Languages and Language Tags</h3>
         
         <p class="definition"><dfn data-lt="well-formed|well-formed language tag|well-formed language tags">Well-formed language tag</dfn>. A language tag that follows the grammar defined in [[BCP47]]. That is, it is structurally correct, consisting of ASCII letters and digit <a>subtags</a> of the prescribed length, separated by hyphens.</p>
         
-        <p class="definition"><dfn data-lt="valid|valid language tag|valid language tags">Valid language tag</dfn>. A language tag that is <a>well-formed</a> and has also been checked to ensure that each of the subtags appears in the <a>IANA Language Subtag Registry</a>.</p>
+        <p class="definition"><dfn data-lt="valid|valid language tag|valid language tags">Valid language tag</dfn>. A language tag that is <a>well-formed</a> and has also been checked to ensure that each of the subtags appears in the <a>IANA Language Subtag Registry</a> and that the <a href="https://tools.ietf.org/html/bcp47#section-2.2.9">conformance criteria for a valid language tag</a> from [[BCP47]] apply.</p>
```

Comment:
I think this is a good idea, but reading back through the conformance critieria (which do add a couple of minor checks--no repeat variants and no repeat extensions), it would be better to do this in reverse, i.e. something like:

> A language tag that is well-formed and which also conforms to the additional [conformance requirements](https://tools.ietf.org/html/bcp47#section-2.2.9) in [BCP47], notably that each of the subtags appears in the IANA Language Subtag Registry.

Reply:
Thanks, that makes sense - done.